projects/aca-shared/src/lib/components/tool-bar/toolbar-menu-item/toolbar-menu-item.component.ts

```diff
@@ -25,9 +25,10 @@
 
 import { Component, Input, ViewEncapsulation } from '@angular/core';
 import { ContentActionRef } from '@alfresco/adf-extensions';
-import { AppExtensionService } from '../../../extensions/extension.service';
+import { AppExtensionService } from '../../../services/app.extension.service';
 
 @Component({
+  // tslint:disable-next-line
```

Comment:
maybe it will make more sense changing the tslint config instead of disabling selector names in each file? like adding "app-" to the allowed rules?